controllers/topolvm_node.go

```diff
@@ -101,7 +140,7 @@ func extractNodeSelectorAndTolerations(lvmCluster lvmv1alpha1.LVMCluster) (*core
 	return nil, nil
 }
 
-func getNodeDaemonSet(lvmCluster *lvmv1alpha1.LVMCluster) *v1.DaemonSet {
+func getNodeDaemonSet(lvmCluster *lvmv1alpha1.LVMCluster, nodeSelector *corev1.NodeSelector, tolerations []corev1.Toleration) *v1.DaemonSet {
```

Comment:
It is not required to pass it in - the extract function can be called inside this function